<?xml version="1.0"?>
<launch>
  <node pkg="rosbag" type="play" name="rosbag" required="true" args="$(find table_exploration)/rosbags/negative_images/6_2022-02-01-19-34-19.bag"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/kuka/camera1/image_raw"/>
  </node>
</launch>